src/api/app/services/scm_initial_status_reporter.rb

```diff
@@ -2,7 +2,7 @@ class ScmInitialStatusReporter < SCMStatusReporter
   attr_accessor :state
 
   def initialize(event_payload, event_subscription_payload, scm_token, workflow_run, event_type = nil)
-    super(event_payload, event_subscription_payload, scm_token)
+    super(event_payload, event_subscription_payload, scm_token, nil, workflow_run.id)
```

Comment:
Hm why pass down `nil` if it's never not `nil`?

Reply:
true